src/services/dashboards/widgets/_components/WidgetFrame.vue

```diff
@@ -199,9 +209,9 @@ export default defineComponent<Props>({
             editModeIconButtonList: computed<IconConfig[]>(() => [
                 {
                     isAvailable: !props.disableExpandIcon,
-                    name: state.isFull ? 'ic_expand-angle' : 'ic_collapse-angle',
+                    name: state.isFull ? 'ic_collapse-angle' : 'ic_expand-angle',
                     handleClick: () => {
-                        emit('click-expand-icon', state.isFull ? 'collapse' : 'expand');
+                        emit('click-expand-icon', state.isFull ? 'collapse' : 'expand', props.widgetIndex);
```

Comment:
I think you can just use `props.isFull`.

Reply:
Emitting `props.widgetIndex` is needed.
I made expanding feature with this sequence.
--- e.g.
If ¡widgetSizeList¡ is given: `['MD', 'MD', **'SM'**, 'LG', 'SM']`,
And user wants to expand `'SM'` to `'FULL'`
-> Each `widgetFrame` has its own `widgetIndex` (in this case, `widgetIndex` is 2)
If expand button has clicked, individual `widgetFrame` emits its own `widgetIndex`.
and `widgetFrameContainer` gains `widgetIndex` which wants to be changed.

So on, `widgetSizeList` is changed by `['MD', 'MD', **'FULL'**, 'LG', 'SM']`
And realign widths to every widget. 

I will add annotations about it. Thanks